tmt/steps/prepare/install.py

```diff
@@ -582,6 +583,90 @@ def install_debuginfo(self) -> None:
             )
 
 
+class InstallBootc(InstallBase):
+    """Install packages using bootc container image mode"""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize bootc installation for package management"""
+        super().__init__(*args, **kwargs)
+
+        self.engine = self.guest.package_manager.engine
+
+    def install_from_repository(self) -> None:
+        self.guest.package_manager.engine.install(
+            *self.list_installables("package", *self.packages),
+            options=Options(
+                excluded_packages=self.exclude,
+                skip_missing=self.skip_missing,
+            ),
+        )
+
+    def install_from_url(self) -> None:
+        self.guest.package_manager.engine.install(
+            *self.list_installables("remote package", *self.remote_packages),
+            options=Options(
+                excluded_packages=self.exclude,
+                skip_missing=self.skip_missing,
+            ),
+        )
+
+    def install_local(self) -> None:
+        # Filelist for packages on the guest
+        filelist = [
+            PackagePath(self.package_directory / filename.name) for filename in self.local_packages
+        ]
+
+        cast(BootcEngine, self.engine).containerfile_directives.append(
+            f'RUN mkdir -p {self.package_directory}'
+        )
+        cast(BootcEngine, self.engine).containerfile_directives.append(
+            f'COPY {" ".join([str(file) for file in filelist])} {self.package_directory}'
```

Comment:
I believe you don't need to materialize a list in `join()`, a generator should work as well: `join(str(file) for file in filelist)`

Reply:
Right, thanks, fixed